genic/params.c

```diff
@@ -87,6 +90,11 @@ void read_parameterfile(char *fname)
     CP.OmegaLambda = param_get_double(ps, "OmegaLambda");
     CP.OmegaBaryon = param_get_double(ps, "OmegaBaryon");
     CP.HubbleParam = param_get_double(ps, "HubbleParam");
+    CP.Omega_fld = param_get_double(ps, "Omega_fld");
+    CP.w0_fld = param_get_double(ps,"w0_fld");
+    CP.wa_fld = param_get_double(ps,"wa_fld");
+    if(CP.OmegaLambda > 0 && CP.Omega_fld > 0)
+        endrun(0, "Cannot have OmegaLambda and Omega_fld (evolving dark energy) at the same time!\n");
```

Comment:
What about "Having OmegaLambda and Omega_fld the same time is highly unusual." Because the code runs just fine.

Reply:
That works too, but doesn't match what CLASS does: class errors. OmegaLambda is really just a special case of Omega_fld so there is no *reason* to have them both at the same time, unless you accidentally used the wrong value in the parameter file.

Reply:
OK then.